data/test-data/org_vote_stat_pa.csv

```diff
@@ -1,34 +1,56 @@
-TBC,1,1,Operating/Program,Fonctionnement/Programme,246327900,234224053,321758616,295850911,303094237,248942508,366807216,290703748,309299504,272955611
-TBC,5,6,Treasury Board Central,Crédit du Conseil du Trésor,749998884,0,750000000,0,750000000,0,635642771,0,530274898,0
-TBC,10,6,Treasury Board Central,Crédit du Conseil du Trésor,2093000,0,103000,0,2090470,0,693000,0,693000,0
-TBC,15,6,Treasury Board Central,Crédit du Conseil du Trésor,0,0,0,0,,,0,0,0,0
-TBC,20,6,Treasury Board Central,Crédit du Conseil du Trésor,2268179661,2186132708,2506134407,2454311704,2719270604,2690624377,2460070604,2406697551,3274680578,3164665468
-TBC,25,6,Treasury Board Central,Crédit du Conseil du Trésor,173453875,0,460111826,0,451599384,0,504827064,0,222997692,0
-TBC,30,6,Treasury Board Central,Crédit du Conseil du Trésor,487698916,0,608191976,0,747466679,0,922128220,0,249426761,0
-TBC,33,6,Treasury Board Central,Crédit du Conseil du Trésor,0,0,137334311,0,108778145,0,73858716,0,42220019,0
-TBC,S,999,Contributions to employee benefit plans,Contributions aux régimes d'avantages sociaux des employés,28996563,28996563,27397562,27397562,26233735,26233735,27019489,27019489,27932725,27932725
-TBC,S,999,Employer contributions made under the Public Service Superannuation Act and other retirement acts and the Employment Insurance Act,Contributions aux régimes d'avantages sociaux des employés,443000000,443000000,444043497,444043497,1162000000,1162000000,339743644,339743644,341250848,341250848
-TBC,S,999,Payments for the Pay Equity Settlement,Paiements au titre de l'entente sur la parité salariale,84322,84322,,,2556,2556,-39653,-39653,-3072,-3072
-TBC,S,999,Payments under the Public Service Pension Adjustment Act,Versements en vertu de la Loi sur la mise au point des pensions du service public,4603,4603,5708,5708,3238,3238,355,355,15823,15823
-TBC,S,999,Salary and Motor Car Allowance,Traitement et allocation pour automobile,78700,78700,80300,80300,82328,82328,83500,83500,84600,84600
-TBC,S,999,Spending of proceeds from the disposal of surplus Crown assets,Dépenses des produits de la vente de biens excédentaires de l'État,3100,0,14852,0,46535,0,41394,,8141,
\ No newline at end of file
+"dept_code","vote_num","vs_type","name_en","name_fr","pa_last_year_5_auth","pa_last_year_5_exp","pa_last_year_5_unlapsed","pa_last_year_4_auth","pa_last_year_4_exp","pa_last_year_4_unlapsed","pa_last_year_3_auth","pa_last_year_3_exp","pa_last_year_3_unlapsed","pa_last_year_2_auth","pa_last_year_2_exp","pa_last_year_2_unlapsed","pa_last_year_auth","pa_last_year_exp","pa_last_year_unlapsed"
+"CBSA",1,1,"Operating/Program","Fonctionnement/Programme",1512906266,1449717812,39351236,1559660142,1421329862,122503987,1757949134.00,1500393332.00,159557676,2051119813,1811958474.00,138252429,1939596099,1746415721,185884855

```

Comment:
CBSA isn't in the test IGOC. Think you wanted CPCC. The snapshot tests specifically queries for CPCC data, so losing it is kind of undermining the snapshot outputs worth.

Side note, I'd recommend making minimal changes whenever updating the test data. We want it to be clear from the snapshot diff that any implementation changes only resulted in the expected output changes. Any time the whole input data/output snapshot changes all at once, it's kind of a loss of confidence in the whole snapshot test itself (unless you then examine the whole snapshot output and manually confirm it).

Fine here because we saw in that earlier snapshot diff that the other fields hadn't changed, just a general caution on snapshot tests.